mint-auditor/src/db/models/gnosis_safe_tx.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use crate::{
+    db::{schema::gnosis_safe_txs, Conn},
+    error::Error,
+    gnosis::api_data_types::RawGnosisTransaction,
+};
+use diesel::prelude::*;
+use serde::{Deserialize, Serialize};
+
+/// Diesel model for the `gnosis_safe_txs` table.
+/// This table stores txs into the monitored gnosis safe.
+#[derive(Debug, Deserialize, Eq, Insertable, PartialEq, Queryable, Serialize)]
+pub struct GnosisSafeTx {
+    /// Ethereum transaction hash.
+    pub eth_tx_hash: String,
+
+    /// The JSON representation of the transaction, as served from the gnosis
+    /// API.
+    pub raw_tx_json: String,
+}
+
+impl GnosisSafeTx {
```

Comment:
The purpose of storing the raw transaction is to give us the option to look at what the response from the API service looked like when we performed the audit. It is currently not being used, but I don't see a reason not to store it in case we want to look at what we got from the Gnosis service at some point in the future.